### PR TITLE
docs: Update 080-090 upgrade docs

### PR DESCRIPTION
**Pull request by OlleLarsson**
**What this PR does / why we need it**:
QA
Updates the upgrade docs and WIP changelog.

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [ ] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.

### CODE REVIEW

File: migration/v0.8.x-v0.9.x/upgrade-apps.md [hunk 1]
```diff
@@ -3,30 +3,42 @@
...
+1. Checkout the new release: `git checkout v0.9.0`.
 
-2. Run init to get new defaults: `./bin/ck8s init`
+2. Update helm to v3.5.0.
+
+3. Optional
+ The default resource requests for Harbor pods have been updated.
+ To get these new values you must remove the current values in your config (`resources:` yaml blocks under `harbor:`) from `sc-config.yaml` before running `./bin/ck8s init` in the next step.
+
+4. Replace the config option `global.environmentName` with `global.clusterName` run `./migration/v0.8.x-v0.9.x/migrate-config.sh`.
```
Comment (Xartos) [inline migration/v0.8.x-v0.9.x/upgrade-apps.md:14]: It seems like the script is an empty file, so I don't think we need to run it.

File: migration/v0.8.x-v0.9.x/upgrade-apps.md [hunk 2]
```diff
@@ -3,30 +3,42 @@
...
     - `user.prometheusPassword`
-    - `externalTrafficPolicy.whitelistRange.prometheus`
-    - `global.dnsPrefix`
 
   - Remove from `wc-config.yaml`
+    - `global.dnsPrefix`
     - `opa.enforcements.imageRegistry`
     - `opa.enforcements.networkPolicies`
     - `opa.enforcements.resources`
+    - `externalTrafficPolicy.whitelistRange.prometheus` wc
```
Comment (Xartos) [inline migration/v0.8.x-v0.9.x/upgrade-apps.md:27]:
```suggestion
    - `externalTrafficPolicy.whitelistRange.prometheus`
```

**Review by Xartos — COMMENTED** [review]
Some small things but otherwise I think it's looking good

File: migration/v0.8.x-v0.9.x/upgrade-apps.md (hunk 1 above)
Comment (OlleLarsson) [reply migration/v0.8.x-v0.9.x/upgrade-apps.md:14]: I have no idea why github says it's empty, it's not :D

Comment (Xartos) [reply migration/v0.8.x-v0.9.x/upgrade-apps.md:14]: Hmm, maybe it's because the diff is only the mode and not a change in the file

**Review by lentzi90 — APPROVED** [review]
Very nice!

**Review by Xartos — APPROVED** [review]

**Pull request merged**